### ISSUE TITLE
When I run ML.Dev.Pred.Category.Sig using template data，I still encountered an error.

### ISSUE THREAD

**Issue by ksjsy** [l-magnificence/Mime#66]
When I run the code with the template data, the system still reports an error.
The code is as follows:
_rm(list = ls())
gc()
setwd("D:/R/program/")
.libPaths("D:/R/packages")

library("Mime1")
library("caret")
library("doSNOW")
load("./External data/Example.ici.Rdata")
load("./External data/genelist.Rdata")

res.ici <- ML.Dev.Pred.Category.Sig(train_data = list_train_vali_Data$training,
                                    list_train_vali_Data = list_train_vali_Data,
                                    candidate_genes = genelist,
                                    methods = c('nb','svmRadialWeights','rf',
                                                'kknn','adaboost','LogitBoost','cancerclass'),
                                    seed = 123,
                                    cores_for_parallel = 60
)
###   60 nodes produced errors; first error: object '.doSnowGlobals' not found

I'm not sure where the problem is. Could you help me take a look? Thank you very much!

**Comment by l-magnificence** [comment]
Have you successfully installed the package? And If your environment don't support multicore, you can run this process in the condition of 1 core, it may run for a long time.

**Comment by upohsdscz** [comment]
> res.ici <- ML.Dev.Pred.Category.Sig(train_data = z$Dataset1,
+                                     list_train_vali_Data = z,
+                                     candidate_genes = genelist,
+                                     methods = c('rf','kknn','nb','adaboost','svmRadialWeights','LogitBoost'),
+                                     seed = 123,
+                                     cores_for_parallel = 10
+ )
---loading the packages ---
---loading the function---
[1] "There existing 11 genes in candidate genes"
[1] "Intersetion of the candidate genes and the colnames of the provided data"
[1] "There existing 11 genes in candidate genes, colnames of training data, colnames of validation data"
Error in na.fail.default(list(Var = c("N", "N", "N", "N", "N", "N", "N",  : 
  missing values in object
In addition: Warning messages:
1: In apply(x[, -c(1:2)], 2, as.numeric) : NAs introduced by coercion
2: In apply(x[, -c(1:2)], 2, as.numeric) : NAs introduced by coercion
3: In apply(x[, -c(1:2)], 2, as.numeric) : NAs introduced by coercion
4: In apply(x[, -c(1:2)], 2, as.numeric) : NAs introduced by coercion
5: In apply(x[, -c(1:2)], 2, as.numeric) : NAs introduced by coercion
6: In apply(x[, -c(1:2)], 2, as.numeric) : NAs introduced by coercion
7: In apply(train_data[, -c(1:2)], 2, as.numeric) :
  NAs introduced by coercion
8: In apply(train_data[, -c(1:2)], 2, as.numeric) :
  NAs introduced by coercion
9: In apply(train_data[, -c(1:2)], 2, as.numeric) :
  NAs introduced by coercion
I also had a problem when running this code 'ML.Dev.Pred.Category.Sig'. 
Could you help me take a look? Thank you very much!
